src/KafkaFlow/Configuration/ConsumerConfiguration.cs

```diff
@@ -1,16 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ConfluentKafka = Confluent.Kafka;
+
 namespace KafkaFlow.Configuration
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Threading.Tasks;
-    using Confluent.Kafka;
-
     internal class ConsumerConfiguration : IConsumerConfiguration
     {
-        private readonly ConsumerConfig consumerConfig;
+        private readonly ConfluentKafka.ConsumerConfig _consumerConfig;
```

Comment:
Why not use the full namespace since the full namespace is almost the same as the alias?

```suggestion
        private readonly Confluent.Kafka.ConsumerConfig _consumerConfig;
```

Reply:
At the time, I had trouble to use the full namespace on my IDE.

I've made the changes. Please check

Reply:
Some of the files where this changed still have `using Confluent.Kafka;` that is not necessary since we are already using the full namespace.

The using can be removed.

Reply:
Done. Please check.